packages/landing/src/components/layer.switcher.dropdown.tsx

```diff
@@ -5,16 +5,17 @@ import { LayerInfo, MapConfig } from '../config.map.js';
 
 type CategoryMap = Map<string, { label: string; options: { label: string; value: string }[] }>;
 
-const CategoryOrder = new Map<string, number>([
-  ['Basemaps', 1],
-  ['Satellite Imagery', 2],
-  ['Urban Aerial Photos', 3],
-  ['Rural Aerial Photos', 4],
-  ['Scanned Aerial Imagery', 5],
-  ['Event', 6],
-  ['Bathymetry', 7],
-  ['Elevation', 8],
-]);
+const Categories = [
+  'Basemaps',
+  'Satellite Imagery',
+  'Urban Aerial Photos',
+  'Rural Aerial Photos',
+  'Scanned Aerial Imagery Basemaps',
+  'Scanned Aerial Imagery',
```

Comment:
What is the different between these two category?

Reply:
Ah, I see you put these two new layers into the new category above all the individual ones.